pkg/resource/plugin/analyzer_plugin.go

```diff
@@ -78,12 +79,25 @@ func NewAnalyzer(host Host, ctx *Context, name tokens.QName) (Analyzer, error) {
 	}, nil
 }
 
-const policyAnalyzerName = "policy"
-
 // NewPolicyAnalyzer boots the nodejs analyzer plugin located at `policyPackpath`
 func NewPolicyAnalyzer(
 	host Host, ctx *Context, name tokens.QName, policyPackPath string, opts *PolicyAnalyzerOptions) (Analyzer, error) {
 
+	proj, err := workspace.LoadPolicyPack(filepath.Join(policyPackPath, "PulumiPolicy.yaml"))
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to load Pulumi policy project located at %q", policyPackPath)
+	}
+
+	var policyAnalyzerName string
+	switch proj.Runtime.Name() {
+	case "nodejs":
+		policyAnalyzerName = "policy"
+	case "python":
+		policyAnalyzerName = "policy-python"
```

Comment:
Having to introduce `pulumi-analyzer-policy-python` is kind of lame, but it's not really user-facing, aside from the fact that we're now distributing an additional `pulumi-analyzer-policy-python` file (`pulumi-analyzer-policy-python.cmd` on Windows).

The alternative would to modify the existing `pulumi-analyzer-policy` script to know whether it should be launching node vs. python (vs. other languages).

Reply:
hmm what if the Policy yaml file had the runtime?

Reply:
The approach taken here is reasonably similar to our language hosts, so I think it's reasonable to use the same approach here, even though it leads to a bit of duplication.  Curious if @pgavlin has strong feelings either way on this.  

For language hosts, I believe we just append the language name to the root and then look for that on the path.  Would doing the same here make this more fundamentally multi-language? (allow a 3rd part to create `pulumi-analyzer-policy-go` themselves?)

Reply:
In fact - would doing just this unblock things like https://github.com/pulumi/pulumi-policy-opa?  Couldn't that project just name itself `pulumi-analyzer-policy-opa` and then mark `runtime: opa` in it's `Pulumipolicy.yaml`?  Users would have to distribute that binary to machines, but actual policies that used it could in principle just work?

Reply:
> Would doing the same here make this more fundamentally multi-language?

> In fact - would doing just this unblock things like https://github.com/pulumi/pulumi-policy-opa? Couldn't that project just name itself `pulumi-analyzer-policy-opa` and then mark `runtime: opa` in it's `Pulumipolicy.yaml`? Users would have to distribute that binary to machines, but actual policies that used it could in principle just work?

Yes, great point! Doing it like this should unblock other languages and OPA.

Reply:
What @lukehoban said. Seems like we should just take the same strategy the language hosts use.